src/app/experiences/page.tsx

```
@@ -41,93 +39,110 @@ export default function ExperiencesPage() {
+                  .includes(option.toLowerCase()) ||
+                experience.location.state
+                  .toLowerCase()
+                  .includes(option.toLowerCase())
             );
           case "Price":
-            if (!experience.price) return false;
-            const priceValue = experience.price.amount;
+            if (!experience.rates) return false;
+            const priceValue = Math.min(
+              ...experience.rates.map((rate) => rate.price.amount)
+            );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)